source/configure/plugins-configuration-settings.rst

```diff
@@ -432,14 +432,19 @@ RTCD service URL
++---------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+| The URL to a running `rtcd <https://github.com/mattermost/rtcd>`__ service instance that will host the calls. | - System Config path: **Plugins > Calls**                                                                                                                 |
+|                                                                                                               | - ``config.json`` setting: ``PluginSettings.Plugins.com.mattermost.calls.rtcdserviceurl``                                                                 |
+|                                                                                                               | - Environment variable: N/A                                                                                                                               |
+| When set (non empty) all the calls will be handled by this external service.                                  |                                                                                                                                                           |
+|                                                                                                               |                                                                                                                                                           |
+| This is an optional field. Changing this setting requires a plugin restart to take effect.                    |                                                                                                                                                           |
++---------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Note**:                                                                                                                                                                                                                                                                 |
+|                                                                                                                                                                                                                                                                           |
+| - The client will self-register the first time it connects to the service and store the authentication key in the database. If no client ID is explicitly provided, the diagnostic ID of the Mattermost installation will be used.                                        |
```

Comment:
We should maybe add a bit about what to do when they get an error (client id already registered or not found) -- I've gotten those from time to time when restarting.

Reply:
I am thinking that would fit more into a troubleshooting section though.

Reply:
Created https://mattermost.atlassian.net/browse/MM-54926 as follow up.